air/src/stack/system_ops/tests.rs

```diff
@@ -96,3 +106,16 @@ pub fn get_assert_test_frame() -> EvaluationFrame<Felt> {
 
     frame
 }
+
+/// Generates the correct current and next rows for the CLK operation and inputs and
+/// returns an EvaluationFrame for testing.
+pub fn get_clk_test_frame(a: u64) -> EvaluationFrame<Felt> {
+    // frame initialised with a fmpupdate operation using it's unique opcode.
```

Comment:
This comment should probably reference `clk` instead of `fmpupdate`

Reply:
agreed , thanks for pointing. I overlooked it when generating this test using `fmpupdate` test.